libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -13,8 +13,19 @@ async function getKey(product) {
   return keyMatch[0]?.key;
 }
 
+async function getECID() {
```

Comment:
Could you re-use alloy to retrieve the ECID?
https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/global-navigation.js#L508

Reply:
YEs, made these changes. Thanks